pylabrobot/resources/plate.py

```diff
@@ -51,7 +51,8 @@ def __init__(
     category: str = "plate",
     lid_height: float = 0,
     with_lid: bool = False,
-    model: Optional[str] = None
+    model: Optional[str] = None,
+    plate_type: Literal["skirted", "semi-skirted", "non-skirted"] = "skirted"
```

Comment:
For semi-skirted plates, is it not important to have the height of the skirt? / do these 3 discrete options capture all required information?

I imagine that for:
- skirted platess: `skirt_height > well_size_z`
- semi-skirted platess: `0 < skirt_height < well_size_z`
- non-skirted platess: `skirt_height = 0`
with `skirt_height` being a float.

Reply:
Yes, you are completely right.

I did not want to change too much about `Plate()` in this PR because it is secondary here to `PlateAdapter`.
That's why I only added this simple 3 word additional information.

I think it would be better to add user information that requires rewriting information in the `Plate()` class in a separate information.
Since here I just add standalone information, and establish that so far all `Plate` instance in PLR have been of "skirted" type, I thought it would be a nice little push to work on it in the future :)

Reply:
Good to know this is a WIP. I shall add a comment